### PR TITLE
feat: improved summary page

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Improved rendering of txn details and summary page

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx [hunk 1]
```diff
@@ -58,10 +58,13 @@ const Success: React.FC<Props> = props => {
...
-              <BuyOrSell orderType={getOrderType(props.order)} />{' '}
-              <FormattedMessage
-                id='modals.simplebuy.order_summary'
-                defaultMessage='Order Summary'
+              <BuyOrSell
+                orderType={orderType}
+                crypto={
+                  orderType === 'BUY'
+                    ? (props.order.outputCurrency as CoinType)
+                    : (props.order.inputCurrency as CoinType)
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx:66]: Can you use `getCoinFromPair(props.order.pair)` instead?

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx]: Done.

File: packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx [hunk 2]
```diff
@@ -116,30 +116,29 @@ const Success: React.FC<Props> = props => {
...
+              <Row>
+                <Title>
+                  <FormattedMessage
+                    id='modals.simplebuy.summary.rate'
+                    defaultMessage='Exchange Rate'
+                  />
+                </Title>
+                <Value data-e2e='sbRate'>
+                  {fiatToString({
+                    unit: counterCurrency as FiatType,
```
Comment (plondon) [inline packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx:129]: unit is always FiatType?

Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/OrderSummary/template.success.tsx:129]: Should be, I'll remove type here

**Pull request merged**